google/cloud/spanner_dbapi/connection.py

```diff
@@ -15,13 +15,24 @@
 
 
 class Connection:
+    """A wrap-around object for the existing :class:`Database` and the corresponding :class:`Instance` objects
```

Comment:
`Database` and `Instance` classes are located in another package, so it's better to write their full paths. Plus, this line doesn't include a period on the end and it's > 100 symbols.

More than that, you don't actually have an `instance` arg here. So, your connection wraps only a database. This comment was actual for another [PR](https://github.com/googleapis/python-spanner-django/pull/481/files), where `instance` arg was added.

Reply:
That description is taken from proposal